cmd/auth-handler.go

```diff
@@ -100,20 +100,20 @@ const (
 
 // Get request authentication type.
 func getRequestAuthType(r *http.Request) authType {
-	if isRequestSignatureV2(r) {
-		return authTypeSignedV2
-	} else if isRequestPresignedSignatureV2(r) {
-		return authTypePresignedV2
+	if isRequestJWT(r) {
```

Comment:
Changing this seems a bit risky, and I don't really see the benefits. All checks seems very cheap.